ci-operator/step-registry/hypershift/agent/create/hostedcluster/hypershift-agent-create-hostedcluster-commands.sh

```diff
@@ -17,6 +17,7 @@ oc extract secret/pull-secret -n openshift-config --to=/tmp --confirm
   --pull-secret=/tmp/.dockerconfigjson \
   --agent-namespace="clusters-${CLUSTER_NAME}" \
   --base-domain=${BASEDOMAIN} \
+  --api-server-address=api.${CLUSTER_NAME}.${BASEDOMAIN} \
```

Comment:
I never tried this, so I just get the IP of the host with the node port. 
I see that you added DNS for:
api.\$HOSTEDCLUSTER_NAME.ostest.test.metalkube.org

Is "ostest.test.metalkube.org" the base domain?

Reply:
yes

Reply:
Ah, maybe we should make some adjustments, currently some hard code